File: CS/WinFormsDashboard_RecalculateTotals/DesignerForm1.cs
namespace WinFormsDashboard_RecalculateTotals {
    public partial class DesignerForm1 : DevExpress.XtraBars.Ribbon.RibbonForm {
        public DesignerForm1() {
            InitializeComponent();
            dashboardDesigner.CreateRibbon();
            dashboardDesigner.LoadDashboard(@"..\..\Dashboards\dashboard1.xml");
            new RecalculateGridTotalsModule().Attach(dashboardDesigner);
        }
    }
}

File: VB/WinFormsDashboard_RecalculateTotals/DesignerForm1.vb
Namespace WinFormsDashboard_RecalculateTotals

    Public Partial Class DesignerForm1
        Inherits DevExpress.XtraBars.Ribbon.RibbonForm

        Public Sub New()
            InitializeComponent()
            dashboardDesigner.CreateRibbon()
            dashboardDesigner.LoadDashboard("..\..\Dashboards\dashboard1.xml")
            Call New RecalculateGridTotalsModule().Attach(dashboardDesigner)
        End Sub
    End Class
End Namespace
